modules/schat.cpp

```diff
@@ -226,9 +230,15 @@ class CSChat : public CModule
 					Table.SetCell("Host", pSock->GetRemoteIP());
 					Table.SetCell("Port", CString(pSock->GetRemotePort()));
 					SSL_SESSION *pSession = pSock->GetSSLSession();
+#ifdef HAVE_OPAQUE_SSL_SESSION
+					if (pSession)
+						Table.SetCell("Cipher", SSL_CIPHER_get_name(SSL_SESSION_get0_cipher(pSession)));
+#else
 					if (pSession && pSession->cipher && pSession->cipher->name)
 						Table.SetCell("Cipher", pSession->cipher->name);
-
+#endif
+					else
+						Table.SetCell("Cipher", "None");
```

Comment:
@DarthGandalf gets to decide the minimum supported version. However, I think that pre-0.9.8 support isn't much of a problem and we could at least try to work there (dunno if it has to be tested explicitly for if it works).